app/helpers/react_components/training_data/dashboard.rb

```diff
@@ -33,6 +29,9 @@ def tracks_request
           options: { stale_time: 0 }
         }
       end
+
+      DEFAULT_STATUS = :untagged
+      private_constant :DEFAULT_STATUS
```

Comment:
@dem4ron Constants tend to go at the bottom of the file if they're private and the top if they're public.